### PR TITLE
Update the contributing document with correct links

### PR DESCRIPTION
**Pull request by jonasrosland**
This will update the contributing document with correct links and descriptions.

Signed-off-by: jonasrosland <jrosland@vmware.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,14 +2,23 @@
...
+* [Bi-weekly public community meetings][community-meetings]
+  * Catch up with [past meetings on YouTube][past-meetings]
+* Chat with us on the CNCF Slack ([get an invite here][cncf-slack] )
+  * [#harbor][users-slack] for end-user discussions
+  * [#harbor-dev][dev-slack] for development of Harbor
+* Want long-form communication instead of Slack? We have two distributions lists:
+  * [harbor-users][users-dl] for end-user discussions
+  * [harbor-dev][dev-dl] for development of Harbor
+
+Follow us on Twitter at [@project_harbor][twitter]
```
Comment (danielpacak) [inline CONTRIBUTING.md:21]: @jonasrosland I think you have to define the [twitter]. Otherwise it's not rendered as a hyperlink

Comment (jonasrosland) [reply CONTRIBUTING.md:21]: It seems I've deleted this branch already so I can't update the PR, do you want me to close this PR and open up a new one?

Comment (danielpacak) [reply CONTRIBUTING.md:21]: In this case I think we can marge this one and fix the Twitter link as a follow up PR

Comment (jonasrosland) [reply CONTRIBUTING.md:21]: Ok, that works for me!

**Comment by michmike** [comment]
i will merge it now.

**Pull request merged**